src/main/kotlin/org/rust/lang/core/types/ty/TySlice.kt

```diff
@@ -18,4 +18,6 @@ data class TySlice(val elementType: Ty) : Ty(elementType.flags) {
         elementType.visitWith(visitor)
 
     override fun toString(): String = tyToString(this)
+
+    override val isCopyable = elementType.isCopyable
```

Comment:
In most cases it wont matter (as slices are typically used behind references) but as far as i can see slices are currently not `Copy` in rust.
https://github.com/rust-lang/rust/issues/30483
```rust
trait T1 {
    type Assoc1: Copy + ?Sized;
}

struct S1 {}

impl T1 for S1 {
    type Assoc1 = [u8];
}
```
During compiling rustc  says `the trait std::marker::Copy is not implemented for [u8]`